docker/build-image.sh

```diff
@@ -17,7 +17,7 @@
 USERNAME=usdotfhwastol
 
 cd "$(dirname "$0")"
-IMAGE="$(./get-image-name.sh)"
```

Comment:
I'm kinda split on this, to be honest. The old script (take a look at get-repo-name.sh) uses the Git repository name as specified by the Git remote URL to handle the case where the folder is renamed or checked out under a different name. Now that there's multiple remote URLs that'll resolve correctly, it doesn't have a ton of benefit over just using the local folder name.